headers/data/arm9.h

```diff
@@ -196,5 +196,10 @@ extern int32_t MUSIC_DURATION_LOOKUP_TABLE_2[128];
 extern sound_lfo_waveform_callback LFO_WAVEFORM_CALLBACKS[16];
 extern bool IS_DISP_ON;
 extern uint32_t GXI_DMA_ID;
+extern struct data_processing_instruction JUICE_BAR_NECTAR_IQ_GAIN;
```

Comment:
Can we make these `const`?

Reply:
Right. I knew I was forgetting someting.